sdk/tables/Azure.Data.Tables/samples/Sample2CreateDeleteEntities.md

```diff
@@ -0,0 +1,95 @@
+# Create and Delete Entities
+This sample demonstrates how to add and delete entities. To get started, you'll need access to either a Storage or Cosmos DB account.
+
+## Create a `TableClient`
+A `TableClient` is needed to perform table-level operations like inserting and deleting entities within the table. There are two ways to get a `TableClient`:
+- Call `GetTableClient` from the `TableServiceClient` with the table name.
+- Create a `TableClient` with a SAS URI, an endpoint and `TableSharedKeyCredential`, or a connection string.
+
+If the table has not been created in the service, call `Create`.
+
+```C# Snippet:TablesSample2CreateTableWithTableClient
+// Construct a new <see cref="TableClient" /> using a <see cref="TableSharedKeyCredential" />.
+var client = new TableClient(
+    tableName,
+    new Uri(storageUri),
+    new TableSharedKeyCredential(accountName, storageAccountKey));
+
+// Create the table in the service.
+client.Create();
+```
+
+//TODO: Include snippet for deleting the table with the TableClient(?)
+
+## Create an entity
```

Comment:
hmm maybe `Create a local entity` ? me, not knowing much, thought this section was about adding the entity to the table